docs/the-new-architecture/pillars-turbomodule.md

```diff
@@ -127,6 +127,9 @@ The shared configuration is a `package.json` file used by yarn when installing y
   "description": "Add numbers with Turbo Native Modules",
   "react-native": "js/index",
   "source": "js/index",
+  "scripts": {
+    "preinstall": "rm -rf node_modules"
```

Comment:
I would refrain from adding this. I don't think we want to recommend wiping `node_modules` as a preinstall step in our docs